docs/agents.md

```diff
@@ -414,6 +414,7 @@ _(This example is complete, it can be run "as is" — you'll need to add `asynci
 You can retrieve usage statistics (tokens, requests, etc.) at any time from the [`AgentRun`][pydantic_ai.agent.AgentRun] object via `agent_run.usage()`. This method returns a [`RunUsage`][pydantic_ai.usage.RunUsage] object containing the usage data.
 
 Once the run finishes, `agent_run.result` becomes a [`AgentRunResult`][pydantic_ai.agent.AgentRunResult] object containing the final output (and related metadata).
+You can inspect [`agent_run.metadata`][pydantic_ai.agent.AgentRun] or [`agent_run.result.metadata`][pydantic_ai.agent.AgentRunResult] after the run completes to read any metadata configured on the agent.
```

Comment:
This deserves a separate section that also explains how to set it. I think much of what's currently in `logfire.md` can be moved here, and then those docs can link here.

Reply:
I put most of the documentation here and only mentioned the relevant parts in `logfire.md`, linking here for details.